support/hydrationStyles.ts

```diff
@@ -0,0 +1,15 @@
+const fs = require("fs");
```

Comment:
@paulcpederson can you use...

```
import fs from "fs";
import path from "path";
```

So we get some type safety?

Reply:
I think we'd have to update our package here :\. Related to https://github.com/Esri/calcite-components/pull/4700#issuecomment-1171487395

Reply:
I tried that and got thrown an error about using import outside a module?

Reply:
ooooh, ok that makes sense, I can make that change once that pr goes in.